system/ee/ExpressionEngine/Service/Validation/Rule/PasswordSecure.php

```diff
@@ -0,0 +1,45 @@
+<?php
+/**
+ * This source file is part of the open source project
+ * ExpressionEngine (https://expressionengine.com)
+ *
+ * @link      https://expressionengine.com/
+ * @copyright Copyright (c) 2003-2021, Packet Tide, LLC (https://www.packettide.com)
+ * @license   https://expressionengine.com/license Licensed under Apache License, Version 2.0
+ */
+
+namespace ExpressionEngine\Service\Validation\Rule;
+
+use ExpressionEngine\Service\Validation\ValidationRule;
+
+/**
+ * Password Validation Rule
+ */
+class PasswordSecure extends ValidationRule
```

Comment:
I'm wondering if the name of this class is misleading... with Password secure.. It's a bit more of password enforce or something along those lines..  I'm thinking we should rename this to be a bit clearer.

enforce password level or something along those liens.

As a validation rule, I feel like we're validating that a password is secure, based on the name.. but they could have the "no" security option chosen and it would pass... 

Any gut reaction on this one @matthewjohns0n or @intoeetive Any good naming ideas?

Reply:
I have no preference here (but also no other ideas).

Reply:
I'd say lets go with enforce password level